debian/control

```diff
@@ -120,7 +120,7 @@ Depends:
   dosfstools,
   grub-efi-amd64-signed [amd64],
   grub-efi-arm64-bin [arm64],
-  mokutil [amd64],
+  mokutil [amd64 arm64],
```

Comment:
We should drop ARCH in this case, `[amd64 arm64]`, isn't it?
```suggestion
  mokutil,
```

Reply:
Ok, after some checks, vyos-1x required mokutil without any condition in the `show version`
Debian also provided mokutil for armhf.
I will modify this.